style.css

```diff
@@ -0,0 +1,89 @@
+/* style sheet for rocket science home */
+
+body {
+        background-image:url("background.png");
+        background-repeat:repeat-y;
+        max-width: 2049px;
+     }
+
+header {
+          text-align:center;
+          border-bottom-style:solid;
+          border-bottom-color:white;
+          border-bottom-width:1px
+       }
+
+header p {
+            width:650px; 
+            display:inline-block;
+            font-family:"Gotham Book", sans-serif;
+            font-size:14pt;
+            color:#FFF;
+         }
+
+section {}
+
+h1 {
+      font-family:"Gotham", sans-serif;
+      font-size:16pt;
+      font-weight:bold;
+      color:#FFF;
+   }
+
+h2 {
+      font-family:"Gotham Book", sans-serif;
+      font-size:14pt;
+      color:#FFF;
+      opacity:0.5;
+   }
+
+#section-title {text-align:center;}
+
+.team-member img {border-radius:50%;}
+
+h3 {
+      font-family:"Gotham", sans-serif;
+      font-size:16pt;
+      font-weight:bold;
+      color:#FFF;
+   }
+
+h4 {
+      font-family:"Gotham Book", sans-serif;
+      font-size:16pt;
+      font-style:italic;
+      color:#FFF;
+      opacity:0.5;
+}
+
+ul {
+      margin:0;padding:0;
+      list-style-type:none;
+      text-align:center;
+   }
+
+ul li { 
+         display:inline-block;
+         max-width:650px;
+         margin:10px
+      }
+
+ul p {
+        max-width:400px;
+        text-align:left;
+        font-family:"Gotham Book", sans-serif;
+        font-size:13pt;
```

Comment:
https://css-tricks.com/css-font-size/#article-header-id-4

Avoid using `pt`! It is often safest to use `px` everywhere.